doc/index.rst

```diff
@@ -5,15 +5,42 @@ Welcome to the DESI Data Model!
 The DESI Data Tree
 ------------------
 
-DESI data are grouped broadly by category, using environment variables to
-define the base directory under which files of that category are kept.
-These variables have a standard location relative to :envvar:`DESI_ROOT`, but code
-uses these variables so that one can swap out different input/output locations
-for testing.
+These pages define the directory structure and file formats for DESI
+data products relative to a root directory :envvar:`DESI_ROOT`.
+Each data release has its own :envvar:`DESI_ROOT`, e.g. the
+`Early Data Release <https://data.desi.lbl.gov/doc/releases/edr/>`_
+at https://data.desi.lbl.gov/public/edr.
+
+Data Directories
+================
+
+Directories under DESI_ROOT
```

Comment:
I would change this line to "Directories under :envvar`DESI_ROOT`:"  The backticks might not come through quite right when writing RST in MD.

Reply:
done